content/configuration/node.md

```diff
@@ -109,9 +120,18 @@ Default: `true`
 
 `boolean | "mock" | "empty"`
 
-Many other Node.js core libraries can be configured as well. See the list of [Node.js core libraries and their polyfills](https://github.com/webpack/node-libs-browser).
+W> This option is only activated (via `NodeSourcePlugin`) when the target is unspecified, "web" or "webworker".
+
+Polyfills for Node.js core libraries from
+[`node-libs-browser`](https://github.com/webpack/node-libs-browser) are used if
+available, when the `NodeSourcePlugin` plugin is enabled.
+See the list of [Node.js core libraries and their polyfills](https://github.com/webpack/node-libs-browser#readme).
```

Comment:
I combined the first three lines that formed one sentence in one line. The last sentence is still separate because the message that it conveys is not depending on the previous line.